lib/postprocess/image.js

```diff
@@ -93,6 +95,19 @@ module.exports = async files => {
           response.headers['content-type'] || response.headers['Content-Type']
         imageExtensions = contentType.split('/')[1]
         imageBuffer = Buffer.from(response.data)
+
+        // check if the file is valid
+        try {
+          sizeOf(imageBuffer)
+        } catch (e) {
+          if (e instanceof RangeError) {
+            const imageSrc = fs.readFileSync(path.join(__dirname, 'missing-image.png')).toString('base64')
+            imageExtensions = 'png'

```

Comment:
Also consider preloading the buffer itself. Encoding base64 is expensive :)

Reply:
Do you mean by simply hardcoding the base64 string?

Reply:
clear now